pkg/bindings/test/images_test.go

```diff
@@ -85,14 +85,16 @@ var _ = Describe("Podman images", func() {
 
 	// Test to validate the remove image api
 	It("remove image", func() {
-		// Remove invalid image should be a 404
+		// NOTE that removing an image that does not exist will still
+		// return a 200 http status.  The response, however, includes
+		// the exit code that podman-remote should exit with.
+		//
+		// The libpod/images/remove endpoint supports batch removal of
+		// images for performance reasons and for hiding the logic of
+		// deciding which exit code to use from the client.
 		response, errs := images.Remove(bt.conn, []string{"foobar5000"}, nil)
 		Expect(len(errs)).To(BeNumerically(">", 0))
-		code, _ := bindings.CheckResponseCode(errs[0])
-		// FIXME FIXME FIXME: #12441: THIS IS BROKEN
-		// FIXME FIXME FIXME: we get msg: "foobar5000: image not known"
-		// FIXME FIXME FIXME: ...with no ResponseCode
-		Expect(code).To(BeNumerically("==", -1))
+		Expect(response.ExitCode).To(BeNumerically("==", -1)) // podman-remote would exit with -1
```

Comment:
Why is the exit code -1? Exit codes should be between 0-255 AFAIK

Reply:
It should be `1` in this case. Tests are also failing.